tgui/packages/tgui/interfaces/common/UserDetails.tsx

```diff
@@ -0,0 +1,42 @@
+import { useBackend } from '../../backend';
+import { Icon, LabeledList, NoticeBox, Section, Stack } from '../../components';
+
+type UserDetailsData = {
+  user: UserData;
+};
+
+type UserData = {
+  name: string;
+  cash: number;
+  job: string;
+  department: string;
+};
+
+export const UserDetails = (props, context) => {
+  const { data } = useBackend<UserDetailsData>(context);
+  const { user } = data;
+
+  if (!user) {
+    return (
+      <NoticeBox>No ID detected! Contact the Head of Personnel.</NoticeBox>
```

Comment:
we dont have a HoP on eris , instead it should be the first officer or your departamental head.